vins_estimator/src/estimator_node.cpp

```diff
@@ -81,31 +51,63 @@ void updateCurrentPositionAndVelocity(const Eigen::Vector3d &imu_linear_accelera
     out_velocity = out_velocity + average_unbiased_accel * dt;
 }
 
+// thread: visual-inertial odometry
 
-void predict(const sensor_msgs::ImuConstPtr &imu_msg)
+class EstimatorNode
 {
-    double t = imu_msg->header.stamp.toSec();
-    if (init_imu)
+public:
+    EstimatorNode()
     {
-        latest_time = t;
-        init_imu = false;
-        return;
+        _nh = ros::NodeHandle("~");
+        // _nh.setParam("config_file", "/home/rosdev/workspace/ros_ws/src/VINS-Mono/config/euroc/euroc_config.yaml");
     }
-    double dt = t - latest_time;
-    latest_time = t;
 
-    linear_acceleration_previous = linear_acceleration_current;
-    angular_velocity_previous = angular_velocity_current;
-    orientation_estimated_previous = orientation_estimated_current;
+    void init()
```

Comment:
Class methods should be in pascal case

source: https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Ok! WIll update all the functions :D